traceevents/src/main/java/com/tomtom/kotlin/traceevents/Tracer.kt

```diff
@@ -329,7 +316,21 @@ public class Tracer private constructor(
             stackTraceHolder = Throwable(),
             eventName = method.name,
             args = args ?: arrayOf(),
-            parameterNames = parameterNames
+            parameterNamesProvider = {
+                /**
+                 * Get the parameter names for the method.
+                 * For performance reasons, make sure this code is normally executed only once per method call. Note
+                 * that this method may be called by multiple threads, so it must be thread-safe as well.
```

Comment:
This documentation is a bit confusing in this place, since 'this method' can refer to the lambda over which we don't have control here.

Reply:
It actually refers to the lamda. Will change `this method` to `this lamda`.

Reply:
Rewrote the documentation.